raytracing/figure.py

```diff
@@ -78,6 +80,32 @@ def addFigureInfo(self, text):
         self.axes.text(0.05, 0.15, text, transform=self.axes.transAxes,
                        fontsize=12, verticalalignment='top', clip_box=self.axes.bbox, clip_on=True)
 
+    def design(self, rayColors: List[Union[str, tuple]] = None, onlyAxialRay: bool = None,
```

Comment:
I like the idea of a `design`.  We should have ready-made groups of parameters wrapped into a design, such as `presentation`, `publication`, `screen`, etc... For instance, publication would avoid colours if possible, no title, whereas presentation could have colours and larger fonts, etc...